__tests__/db.test.js

```diff
@@ -213,6 +213,19 @@ describe('App.js', () => {
           .delete('/api/comments/1')
           .expect(204)
         })
+        test('GET - /api/users - responds with 200 & users array', () => {
+          return request(app)
+          .get('/api/users')
+          .expect(200)
+          .then((res) => {
+            expect(res.body.length).toEqual(4)
+            expect(res.body[0]).toMatchObject({
+              username: 'butter_bridge',
+              name: 'jonny',
+              avatar_url: 'https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg'
+            })
```

Comment:
Instead of checking just the first user, I would check that every user has a username, name and avatar url and what data types they are. ForEach could be handy 👍